### PR TITLE
OC-9482 Participant Matrix does not have correct Form Event Action Links	

### PR DESCRIPTION
**Pull request by rooney**
https://jira.openclinica.com/browse/OC-9482

### CODE REVIEW

**Comment by svadla-oc** [comment]
jenkins retest this please

File: src/main/java/org/akaza/openclinica/control/managestudy/EventCrfLayerBuilder.java [hunk 1]
```diff
@@ -575,6 +557,32 @@ private void removeEventCrf(HtmlBuilder builder, EventCRFBean eventCrf, StudySub
...
+        } else {
+            formLayoutId = eventCrf.getFormLayoutId();
+        }
+        String href = "pages/managestudy/chooseCRFVersion?crfId=" + crf.getId() + "&crfName=" + crf.getName() + "&formLayoutId=" + Integer.toString(formLayoutId) + "&formLayoutName=" + Integer.toString(formLayoutId) + "&studySubjectLabel=" + studySubject.getLabel() + "&studySubjectId=" + studySubject.getId() + "&eventCrfId=" + eventCrf.getId() + "&eventDefinitionCRFId=" + eventDefinitionCrf.getId() + "&originatingPage=ListStudySubjects";
+        builder.a().append(" class=\"accessCheck\"  ").href(href).close();
+        builder.append("<span border=\"0\" align=\"left\" class=\"icon icon-icon-reassign3\"/>");
+        builder.aEnd();
+    }
+
+    private void reassignEventCrf(HtmlBuilder builder, EventDefinitionCRFBean eventDefinitionCrf, EventCRFBean eventCrf, CRFBean crf, StudySubjectBean studySubject, String link) {
```
Comment (svadla-oc) [inline src/main/java/org/akaza/openclinica/control/managestudy/EventCrfLayerBuilder.java]: If the only difference between this method and the one above is the link, can you refactor these methods into one and handle the link when it is present?

Comment (rooney) [reply src/main/java/org/akaza/openclinica/control/managestudy/EventCrfLayerBuilder.java]: @svadla-oc OK refactored

Comment (rooney) [reply src/main/java/org/akaza/openclinica/control/managestudy/EventCrfLayerBuilder.java]: https://github.com/OpenClinica/OpenClinica/pull/2586/commits/5826feb7aaf1f5155ee5dced843e0e44983ceeec

**Comment by svadla-oc** [comment]
jenkins retest this please

**Comment by svadla-oc** [comment]
👍

**Pull request merged**